sdk/core/core/src/az_auth.c

```diff
@@ -204,8 +205,12 @@ AZ_NODISCARD az_result az_auth_get_token(
     AZ_RETURN_IF_FAILED(az_http_client_send_request_and_get_body(&hrb, &response_buf));
   }
 
-  AZ_RETURN_IF_FAILED(az_json_get_object_member_string(
-      az_mut_span_to_span(response_buf), AZ_STR("access_token"), out_result));
+  {
+    az_json_value value;
+    AZ_RETURN_IF_FAILED(az_json_get_object_member(
+        az_mut_span_to_span(response_buf), AZ_STR("access_token"), &value));
```

Comment:
Would "access_token" still work, or do I need to specify it as "/access_token" ?

Reply:
It will still work for this function. I'll have another PR for `az_json_get_by_pointer` function.